RunScriptHelper.xcodeproj/project.pbxproj

```diff
@@ -62,6 +62,7 @@
 /* Begin PBXShellScriptBuildPhase section */
 		34E5ABCB24620DE40070A80B /* SwiftLint Script */ = {
 			isa = PBXShellScriptBuildPhase;
+			alwaysOutOfDate = 1;
```

Comment:
This removes the same warning we saw in the main project. We run swiftlint on every build so we don't need to have dependency analysis turned on

Reply:
The one from SwiftLint where Xcode would say "do you know you're running this script on all builds?"